xbmc/GUIInfoManager.cpp

```diff
@@ -1705,7 +1705,9 @@ const infomap system_labels[] =  {{ "hasnetwork",       SYSTEM_ETHERNET_LINK_ACT
                                   { "hascms",           SYSTEM_HAS_CMS },
                                   { "privacypolicy",    SYSTEM_PRIVACY_POLICY },
                                   { "haspvraddon",      SYSTEM_HAS_PVR_ADDON },
-                                  { "supportscpuusage", SYSTEM_SUPPORTS_CPU_USAGE }};
+                                  { "supportscpuusage", SYSTEM_SUPPORTS_CPU_USAGE },
+                                  { "windowshdroff",    SYSTEM_WINDOWS_HDR_OFF },
```

Comment:
Can these be made more generic and not be glued completely to a platform? I would prefer to have simply `hdron` and `hdroff` even if the returned value is only meaningful when using windows. Please also add the documentation above the array like the other infolabels

Reply:
Done in https://github.com/xbmc/xbmc/pull/17012/commits/fae9ab8d6f0f3e5be64b4b108c8eb154f2088785

Reply:
Thanks!